specification/at_protocol_specification.md

```diff
@@ -1,37 +1,33 @@
 # atProtocol Specification
 
-\
-
-| **Subject**   | atProtocol specification|
-| ------------- | ------------------------|
+| **Subject**   | atProtocol specification                           |
+| ------------- | -------------------------------------------------- |
 | **Author(s)** | Colin Constable, Kevin Nickels, Jagannadh Vanghuri |
-| | Gary Casey, Chris Swan |
-| **Revision**  | v0.2.1 (draft) |
-| **Date**      | Dec, 5, 2023  |
-
-\
+|               | Gary Casey, Chris Swan, Xavier Chanthavong         |
+| **Revision**  | v1.0.0 (draft)                                     |
```

Comment:
Are we okay that I am keeping this marked as a draft version?

Reply:
Yes ... we should schedule a read-through on the architecture call some day to gather final edits, catch more things we missed etc and do another PR for those, at which point I think we can remove the `(draft)` annotation